app/tasks/maintenance_tasks/array_task.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module MaintenanceTasks
+  class ActiveRecordTask < Task
```

Comment:
This is a copy-paste mistake 😅 
```suggestion
  class ArrayTask < Task
```